src/background/extension-details-view-controller.ts

```diff
@@ -27,7 +27,9 @@ export class ExtensionDetailsViewController implements DetailsViewController {
 
         const tab = await this.browserAdapter.createTabInNewWindow(this.getDetailsUrl(targetTabId));
 
-        this.tabIdToDetailsViewMap[targetTabId] = tab.id;
+        if (tab?.id != null) {
```

Comment:
I think it would make more sense for this case to throw an error explicitly; if we aren't able to assign the tab-to-view map entry, it means that the controller is in a bad state and isn't safe to use in future calls.